src/java.base/share/classes/com/sun/crypto/provider/SunJCE.java

```diff
@@ -460,27 +460,13 @@ void putEntries() {
         /*
          * Key Derivation engines
          */
-        ps("KDF", "HKDF/HmacSHA256",
+        ps("KDF", "HkdfSHA256",
                 "com.sun.crypto.provider.HkdfKeyDerivation$HkdfSHA256");
-        ps("KDF", "HKDF/HmacSHA384",
+        ps("KDF", "HkdfSHA384",
                 "com.sun.crypto.provider.HkdfKeyDerivation$HkdfSHA384");
-        ps("KDF", "HKDF/HmacSHA512",
+        ps("KDF", "HkdfSHA512",
```

Comment:
In fact I like the old name better. At least I think the `HKDF` should be all capitals. Whether you need the `Hmac` word and what delimiter to use can be discussed.

Reply:
What about `HKDFWithHmacSHA256`? This would be consistent with previous PBE names.